grml2usb.8.txt

```diff
@@ -68,14 +68,15 @@ Avoid executing commands, instead show what would be executed.
 Warning: please notice that the ISO has to be mounted anyway, otherwise
 identifying the Grml flavour would not be possible.
 
-  *--fat16*::
+  *--format*::
 
-Format specified partition with FAT16.
+Format specified partition with VFAT.
```

Comment:
tbh I think VFAT has no meaning outside the linux kernel and `mkfs.vfat`. Should just say FAT here (when referring to the filesystem; when referring to `mkfs.vfat` or the kernel driver its appropriate to say `vfat`)